aries-site/src/pages/whats-new.mdx

```diff
@@ -52,6 +52,84 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
+
+### Stay current with the latest changes
+- **Voice and Tone** - Updates in progress for Voice & Tone page.
+- **404 Page** - Designed a new 404 page for our Design System site.
+- **Accessibility** - Updates in progress for Accessibility page to provide more resources and guidance.
+- **SideBar** - Removed sidebar from examples and documentation, this component is going to be deprecated by the end of the year. 
+
+### Latest Releases
+- **Grommet v2.20.0** - Release includes the NEW Tag component along with enhancements to Calendar, Carousel, DateInput, Form, Video; fixes applied to Anchor, Button, Calendar, Clock, Select, Text, and TextInput. Full details about this release are available in the [Release Notes.](https://github.com/grommet/grommet/releases/tag/v2.20.0)
+
+## December 24 - January 3, 2022 Happy Holidays 🎉
+
+## December 20 - 23, 2021
+
+### "Big Things" progress updates
+Here are the latest developments regarding roadmap items:
+- **Page Layouts** - Commenced explorations on [Dashboards within Page Layouts.](https://www.figma.com/file/4rdBkUlHd5MCVq3hvUOXHd/Page-Layouts?node-id=2159%3A17698)
+- **NameValueList** - Commenced ideations on editable horizontal form fields with NameValue lists in [Figma](https://www.figma.com/file/Ny7rYK5gI5VO4PtZsq6vGC/HPE-Name-Value-List?node-id=798%3A2829) and published on the [Design System site](https://design-system.hpe.design/components/namevaluelist?q=name#name-value-pair-as-a-read-only-display-for-text-inputs)
+- **Navigation** - Commenced research for [Ascending Navigation.](https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=2382%3A13827)
+
+### Stay current with the latest changes
+- **Toast Notifications** - Updated guidance on [Toast Notifications](https://design-system.hpe.design/templates/toast-notifications?q=toast)
+
+## December 6 - December 17, 2021
+
+### "Big Things" progress updates
+Here are the latest developments regarding roadmap items:
+- **Button** 
+   - Completed guidance on placement of the Cancel Button in a Form and Modal in [Figma]( https://www.figma.com/file/Oi5UEEQ33VCAyOKQcZ8Nr8/HPE-Button-Component?node-id=667%3A385) and published on the [Design System site](https://design-system.hpe.design/components/button?q=but#styling)
+   - Updated Figma with the [new hover states for Primary and Color Buttons.](https://www.figma.com/file/Oi5UEEQ33VCAyOKQcZ8Nr8/HPE-Button-Component)
+   - Updated Button Figma file on Cancel Buttons following [default styling in Forms and Modals.](https://www.figma.com/file/Oi5UEEQ33VCAyOKQcZ8Nr8/HPE-Button-Component?node-id=4688%3A9756)
+- **Tag** 
+   - Commenced [explorations for Tag component.](https://www.figma.com/file/8zrInotU4JK9LWP4WyZy25/HPE-Design-System-Tags?node-id=2%3A3)
+   - Completed [ideations for Tags component.](https://www.figma.com/file/8zrInotU4JK9LWP4WyZy25/HPE-Design-System-Tags?node-id=286%3A2375)
```

Comment:
How did we "commence exploration" but "complete ideation" ? Feels like exploration comes first then ideation?

Reply:
Tagging @vavalos5 just to clarify on her end